src/views/Shipment.vue

```diff
@@ -110,19 +126,61 @@ export default defineComponent({
     },
     async completeShipment() {
       const alert = await alertController.create({
-        cssClass: "my-custom-class",
         header: "Complete Shipment",
         message:
-          "Make sure you have entered the correct quantities for each item before proceeding.",
-        buttons: ["Cancel", "Complete"],
+          (translate("Make sure you have entered the correct quantities for each item before proceeding.")),
+        buttons: [
+          {
+            text: this.$t("Cancel"),
+            role: 'cancel',
+          }, 
+          {
+            text:this.$t('Complete'),
+            handler: () => {
+              this.updateShipment();
+            },
+          }
+        ],
       });
       return alert.present();
     },
+    async updateShipment() {
+      this.items.items.filter((item: any) => {
+        if(item.quantityAccepted > 0) {
+          const payload = {
+            shipmentId: this.items.shipmentId,
+            facilityId: this.user.facilityId,
+            shipmentItemSeqId: item.itemSeqId,
+            productId: item.productId,
+            quantityAccepted: item.quantityAccepted,
+            locationSeqId: this.items.locationSeqId
+          }
+          this.store.dispatch('shipment/updateShipmentProducts', payload).then(() => {
```

Comment:
We will pass items here and payload will be prepared in action.

Reply:
Done